eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -15,14 +16,28 @@ steps:
 
   - ${{ each repo in parameters.Repos }}:
     - pwsh: |
+        Write-Host "Patch 1 $(PatchFilesLocation)"
         git clone --branch ${{ parameters.BaseBranchName }} https://github.com/azure/${{ repo }}
         $repoPath = "${{ repo }}/${{ parameters.DirectoryToSync }}"
-
-        if (!(Test-Path $repoPath)) { mkdir $repoPath }
-        Remove-Item -v -r $repoPath
-        Copy-Item -v -r $(Build.SourcesDirectory)/${{ parameters.DirectoryToSync }} $repoPath
-        Get-ChildItem -r $repoPath
-
+        if ('$(PatchFilesLocation)')
```

Comment:
Should this be a file exists check?

Reply:
The variable is only set if patch files were created so just checking if it exists is sufficient.

Reply:
I guess we can do with just an exists check instead of a file exists